kms/spec/kms_spec.rb

```diff
@@ -0,0 +1,367 @@
+# Copyright 2017 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rspec"
+require "tempfile"
+require "google/apis/cloudkms_v1beta1"
+require_relative "../kms"
+
+describe "Key Management Service" do
+
+  def create_service_client
+    kms_client = Google::Apis::CloudkmsV1beta1::CloudKMSService.new
+    kms_client.authorization = Google::Auth.get_application_default(
+      "https://www.googleapis.com/auth/cloud-platform"
+    )
+    kms_client
+  end
+
+  def test_create_keyring project_id:, key_ring_id:, location:
```

Comment:
When I first read this, I wondered if you were using [minitest](http://docs.seattlerb.org/minitest/#label-Unit+tests) because of the [`def test_something`](http://docs.seattlerb.org/minitest/#label-Unit+tests) method.

Then I guessed that, because you prefixed the method with `test_` that maybe you were doing *tons* of identical assertions so you refactored them into a method.  I read `test_create_keyring` and I think I should pass my key ring to this method and it will test it for me.

**tl;dr** Rename these, eg. `create_test_keyring` (similar to `create_service_client`)